test/test_forward_index_builder.cpp

```diff
@@ -157,6 +167,7 @@ void write_batch(std::string const &basename,
 
 TEST_CASE("Merge forward index batches", "[parsing][forward_index]")
 {
+    tbb::task_scheduler_init init(2);
```

Comment:
I don't think we even need to pass a number here, though it probably doesn't matter.